README.md

```diff
@@ -0,0 +1,900 @@
+
+Powerline delegations MUST NOT be used as the root delegation to a resource. A priori there is no such thing as a `null` subject a prior.
+ 
+A very common use case for Powerlines is providing a stable DID across multiple agents (e.g. representing a user with multiple devices). This enables the automatic sharing of authority across their devices without needing to share keys or set up a threshold scheme. It is also flexible, since a Powerline delegation MAY be [revoked][revocation].
+
+``` mermaid
+sequenceDiagram
+    autonumber
+
+    participant Email Server
+    participant Alice Root
+    participant Alice's Phone
+    participant Alice's Tablet
+    participant Alice's Laptop
+
+    Alice Root ->> Alice's Phone: Delegate {sub: null, cmd: "/"}
+    Alice Root ->> Alice's Tablet: Delegate {sub: null, cmd: "/"}
+    Alice Root ->> Alice's Laptop: Delegate {sub: null, cmd: "/"}
+
+    Email Server ->> Alice Root: Delegate {sub: "did:example:email", cmd: "/msg/send"}
+
+    Alice's Tablet -->> Email Server: INVOKE! {sub: "did:example:email", cmd: "/msg/send", proofs: [❹,❷]}
+```
+
+Powerlines MAY include other restrictions, such as [time bounds][Time Bounds], [Commands][Command], and [Policies][Policy]. For example, the ability to automatically redelegate read-only access to arbitrary CRUD resources could be expressed as:
+
+``` js
+{
+  "iss": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp",
+  "aud": "did:key:zQ3shokFTS3brHcDQrn82RUDfCZESWL1ZdCEJwekUDPQiYBme",
+  "sub": null, // 👈 ⚡ Powerline
+  "cmd": "/crud/read",
+  "pol": [],
+  // ...
+}
+```
+
+## Command
+[Command]: #command
+
+The [Command][High Level Command] MUST be a `/` delimited path describing set of commands delegated.
+Delegation covers exact [Command] specified and all the commands described by a paths nested under that specified command. 
+
+> [!NOTE]
+> The command path syntax is designed to support forward compatible protocol extensions. Backwards-compatibl️️️️️️️️️️e capabilities MAY be introduced as command subpaths.
+
+> [!WARNING]
+> By definition `"/"` implies all of the commands available on a resource, and SHOULD be used with great care.
+
+# Policy
+[Policy]: #policy
+
+UCAN Delegation uses predicate logic statements extended with [jq]-inspired selectors as a policy language. Policies are syntactically driven, and MUST constrain the `args` field of an eventual [Invocation].
+
+A Policy is always given as an array of predicates. This top-level array is implicitly treated as a logical `and`, where `args` MUST pass validation of every top-level predicate.
+
+Policies are structured as trees. With the exception of subtrees under `any`, `or`, and `not`, every leaf MUST evaluate to `true`. 
+
+A Policy is an array of statements. Every statement MUST take the form `[operator, selector, argument]` except for negation which MUST take the form `["not", statement]`.
```

Comment:
Unless I'm missing something, `and` and `any` also have that 2-tuple form.

Reply:
answering to myself: `and` and `or` can be *both* connectives (2-tuple) or quantifier (3-tuple).


I didn't see that one coming ;)

Reply:
Hmm, those two are actually N-tuple, with N >= 2

Reply:
No, they are 2-tuple, but the ABNF grammar is wrong